nautobot/docs/development/extending-models.md

```diff
@@ -41,19 +41,17 @@ Add the name of the new field to `csv_headers` and included a CSV-friendly repre
 
 ## 4. Update relevant querysets
 
-<!-- v2 TODO(jathan): Replace prefetch_related with select_related -->
-
-If you're adding a relational field (e.g. `ForeignKey`) and intend to include the data when retrieving a list of objects, be sure to include the field using `prefetch_related()` as appropriate. This will optimize the view and avoid extraneous database queries.
+If you're adding a relational field (e.g. `ForeignKey`) and intend to include the data when retrieving a list of objects, be sure to include the field using `select_related()` and/or `prefetch_related()` as appropriate. This will optimize the view and avoid extraneous database queries.
```

Comment:
This is great improvement, but without cacheops it's not telling the entire story.

- `select_related` for `ForeignKey` forward lookups
- `prefetch_related` for `ManyToManyField` forward lookups, and reverse lookups of `ForeignKey` relationships.

Reply:
```suggestion
If you're adding a relational field (e.g. `ForeignKey`) and intend to include the data when retrieving a list of objects, be sure to include the field using `select_related()` (for `ForeignKey` forward lookups) and/or `prefetch_related()` (for `ForeignKey` reverse lookups and `ManyToManyField` lookups) as appropriate. This will optimize the view and avoid extraneous database queries.
```

Reply:
:shipit: